nearlib/exports.js

```diff
@@ -12,5 +13,6 @@ nearLib.NearClient = NearClient;
 nearLib.Account = Account;
 nearLib.BrowserLocalStorageKeystore = BrowserLocalStorageKeystore;
 nearLib.LocalNodeConnection = LocalNodeConnection;
+nearLib.KeyPair = KeyPair;
```

Comment:
`nearLib` contests should ideally be assembled in `index.js`, so that you can just do `require('nearlib')` and get access to all objects, not just for `browserify` use case

Reply:
agree, a lot of infra around nearlib needs to be consolidated.